textdb/textdb-sandbox/JinggangHelloWorld.java

```diff
@@ -1,6 +0,0 @@
-#include<iostream>
-public class JinggangHelloWorld{
-	public static void main(String[] args){
-		cout << "Hello World!" << endl;
```

Comment:
@kishore-narendran Some students are not using Mac, and I prefer **not** to include platform-dependent files. Can we remove it?